src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/db/Hearing.java

```diff
@@ -65,6 +65,11 @@
 
     private Duration duration;
 
+    private Integer numberOfSessions;
+
+    @Column(name = "is_multisession")
```

Comment:
is it only me or another _ would be better: is_multi_session ?

Reply:
technically the English is 'multi-session' and not 'multi session', which the _ would suggest :D